docs/api/LightroomPartnerAPIsSpec.json

```diff
@@ -1110,6 +1110,116 @@
             "$ref":"#/components/responses/ServiceUnavailable"
           }
         }
+      },
+      "post": {
+        "tags": [
+          "albums"
+        ],
+        "summary": "Update album",
+        "description": "Update an existing album.",
```

Comment:
Good to add this info: the existing album should be self created (serviceId check) and of project and project set subtypes.